Makefile

```diff
@@ -18,8 +19,8 @@ clean:
 # remove all files that are ignored by git
 deep-clean:
 	@make clean
-	@find . -type d -name 'node_modules' -exec rm -r {} + &> /dev/null
-	@find . -type d -name 'dist' -exec rm -r {} + &> /dev/null
+	@find . -type d -name 'node_modules' -exec rm -r {} + 2> /dev/null
```

Comment:
I'm in favor of remove these 2 find commands entirely, they're very verbose and unless you have a PhD in stupid command line functions then you won't know what it's doing. Just update the rm command to include those folders

```make
@rm -rf \
		node_modules packages/*/node_modules \
		dist packages/*/dist \
		.nyc_output \
		npm-debug.log \
		yarn-error.log
```